lib/msf/base/config.rb

```diff
@@ -257,6 +257,13 @@ def self.save(opts)
     self.new.save(opts)
   end
 
+  # Deletes all config options for saved module options.
+  # 'framework/' config options are not deleted as they contain important instance-specific information.
+  # @return [void]
+  def self.delete_all
```

Comment:
This seems like a misnomer, as it doesn't delete everything. Potentially `delete_module_config` or similar?

Reply:
Maybe this code should just live in the `cmd_save` class entirely, as I don't know who would reuse this functionality in the future directly on the config object like this 🤔

Reply:
Changed in the latest commit 👍